util/Migrator/DbScripts/2021-06-08_00_EntityFrameworkSupport.sql

```diff
@@ -251,6 +251,7 @@ BEGIN
         @RevisionDate
```

Comment:
Given the latent date this will be merged, maybe time to rename this file with a new date/sequence?

Reply:
Changed to today's date